api/openapi_server/test/__init__.py

```diff
@@ -33,9 +34,11 @@ def create_app(self):
         database.DataAccessLayer.db_init()
 
         logging.getLogger('connexion.operation').setLevel('ERROR')
-        app = connexion.App(__name__, specification_dir='../_spec/')
+        app = connexion.App(__name__)
         app.app.json_encoder = JSONEncoder
-        app.add_api('openapi.yaml', pythonic_params=True)
+        app.add_api(get_bundled_specs(Path('openapi_server/openapi/openapi.yaml')),
+                arguments={'title': 'Home Unite Us'},
+                pythonic_params=True)   
```

Comment:
This code is duplicated from `__main__.py`. Have we thought about add an `App` class to encapsulate this logic? If we did it that way then `__main__.py` and `BaseTestCase` would just need to create an instance of our app class, instead of replicating this kind of complex logic. 

If we go that route I'd probably make `get_bundled_specs` a "private" method within the app class.

Reply:
You can choose a better name than just `App`, of course

Reply:
@Joshua-Douglas @erikguntner I can make that change in the 465-deploy-to-ec2 after this PR goes to main.

Reply:
@paulespinosa Sounds good to me, thanks!